platforms/jvm/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CodeNarcPlugin.java

```diff
@@ -48,8 +48,10 @@
 public abstract class CodeNarcPlugin extends AbstractCodeQualityPlugin<CodeNarc> {
 
     public static final String DEFAULT_CODENARC_VERSION = appropriateCodeNarcVersion();
+    private static final String CONFIG_FILE_PATH = "config/codenarc/codenarc.xml";
```

Comment:
💅 Would it make sense to prefix it with `DEFAULT_`?

Reply:
Makes sense yes, fixed